server/graphql/v2/mutation/VirtualCardMutations.ts

```diff
@@ -58,6 +58,53 @@ const virtualCardMutations = {
       });
     },
   },
+  editVirtualCard: {
+    description: 'Edit existing Virtual Card information',
+    type: new GraphQLNonNull(VirtualCard),
+    args: {
+      virtualCard: {
+        type: new GraphQLNonNull(VirtualCardEditInput),
+        description: 'Virtual Card being edited its new values',
+      },
+      userAccount: {
+        type: AccountReferenceInput,
+        description: 'The user account reference to attach the Virtual Card to if desired',
+      },
+    },
+    async resolve(_: void, args, req: express.Request): Promise<VirtualCardModel> {
+      if (!req.remoteUser) {
+        throw new Unauthorized('You need to be logged in to assign a virtual card');
+      }
+
+      const { expireDate, cvv } = args.virtualCard.privateData;
+      if (!expireDate || !cvv) {
+        throw new BadRequest('VirtualCard missing cardNumber, expireDate and/or cvv', undefined, {
+          expireDate: !expireDate && 'Expire Date is required',
+          cvv: !cvv && 'CVV is required',
+        });
+      }
+
+      const virtualCard = await models.VirtualCard.findOne({ where: { id: args.virtualCard.id } });
+      if (!virtualCard) {
+        throw new NotFound('Could not find Virtual Card');
+      }
+      if (!req.remoteUser.isAdmin(virtualCard.HostCollectiveId)) {
+        throw new Unauthorized("You don't have permission to edit this collective");
+      }
+      const UserId =
+        args.userAccount &&
+        (
+          await fetchAccountWithReference(args.userAccount, {
```

Comment:
Check that the account can be fetched first and throw an error if it doesn't.

Check that's it's an Individual account (`Collective type=USER`) and throw an error if it doesn't.